wpmcpp/src/mainwindow.cpp

```diff
@@ -156,16 +156,7 @@ MainWindow::MainWindow(QWidget *parent) :
 
     // Npackd runs elevated and the taskbar does not. We have to allow the
     // taskbar event here.
-    HINSTANCE hInstLib = LoadLibraryA("USER32.DLL");

```

Comment:
Because ChangeWindowMessageFilterEx can be accessed directly if the program links user32.lib. If it is not working with mingw, I could change it.